docs/eps/PIPELINE.md

```diff
@@ -0,0 +1,122 @@
+# Pipeline CI/CD
+
+## Histórico de Revisões
+|Data|Versão|Descrição|Autor|
+| --- | --- | --- | --- |
+|06/05/2018|0.0.1|Versão inicial do documento do pipeline|João Pedro Sconetto e Mariana Mendes|
+
+## Processos de DevOps
+O presente documento visa esclarecer e registrar todos os processos, a cultura implementada e os padrões que foram utilizados no projeto __Dr. Down__ no viés das práticas de DevOps a fim de unificar o desenvolvimento e as operações inerentes do projeto supracitado.
+
+O documento se divide em duas partes:
+
+* Integração Contínua (CI - _Continuous Integration_)
+
+* Deploy Contínuo (CD - _Continuous Deploy_)
+
+Onde a primeira parte irá mostrar as técnicas e itens aplicados ao projeto no propósito, como o nome da técnica diz, integrar continuamente o trabalho desenvolvido pela equipe. A segunda parte irá mostrar as técnincas e itens aplicados ao projeto com o objetivo de fazer a implantação contínua do projeto na infraestrutura utilizada pela equipe, de forma automatizada.
+
+### Integração Contínua:
+
+#### Padrão de _Commit_
+
+##### Por questões de padronização documentamos o seguinte estilo de _commit_:
+
+* Os _commits_ devem ser todos em __inglês__;
+
+* Ele deve conter um título curto e objetivo do que foi feito naquele _commit_;
+
+* Após esse título, deve-se descrever, com um pouco mais de detalhes, todas as atividades executadas.
+
+* Caso esteja trabalhando em com algum associado assine nos seus _commits_ os seus parceiros
+
+__Exemplo:__
+
+    Creating project community files (Título curto e objetivo)
+
+    Adds project license (Descrição de uma das atividades)
+
+    Adds project code of conduct file
+
+    Adds project contributing file
+
+    Adds project issue template file
+
+    Adds projects pull request file
+
+    Co-authored-by: John Doe <john@email.com> (Assinatura de parceria)
+
+#### Política de _Branchs_
+
+Tendo como meta manter a integralidade e confiabilidade do código do projeto foi proposta a utilização de política de branches.
+Essa Política de Branches deverá guiar os desenvolvedores na forma de organização de suas contribuições ao projeto.
+
+__OBS__: A política de _branchs_ foi idealizada para trabalhar em conjunto com a ferramenta do _git flow_, sua documentação e mais informações podem ser acessadas [aqui](https://github.com/nvie/gitflow).
+
+* __master__ - Branch principal do repositório onde será permitida somente a integração de software consolidado e testado. Essa branch será exclusiva para a entrega de Realeases, ou seja, um conjunto maior de funcionalidades que integram o software, aqui estará a versão _**stable**_ do software.
+
+* __develop__ - Branch para integração de novas funcionalidades, onde será permitido a entrega das features desenvolvidas e que estão em um estágio avançado de completude. Será o branch base para o início do desenvolvimento das features e da correção de bugs. Aqui também serão _mergeadas_ as releases.  
```

Comment:
"__develop__ - Branch para integração de novas funcionalidades, ~~onde~~ __na qual__ será ~~permitido~~ __permitida_ a entrega das features desenvolvidas e que estão em um estágio avançado de completude. Será ~~o~~ __a__ branch base para o início do desenvolvimento das features e da correção de bugs. Aqui também serão "mergeadas" as releases."